README.md

```diff
@@ -1,35 +1,59 @@
 # community-ninja-board
 
-Front end user interface for the Giveback Ninja Program 
+Front end user interface for the Giveback Ninja Program(v1) and a parallel system to replace the former called v2.
 
-## Deployment locally (laptop) / or for DEV purposes 
+## Deployment locally (laptop) / or for DEV purposes
 
-### Setting up the environment variables
+### Local v1 Deployment
+
+#### Setting up the environment variables
 ```
 export TRELLO_API_TOKEN=<your token>
 export TRELLO_API_KEY=<your token>
 export GITHUB_API_TOKEN=<your token>
 export GITLAB_API_TOKEN=<your token>
 export GD_CREDENTIALS=<contents of you google drive credentials.json file (escaped & in quotes)>
-export GRAPHS_PROXY=<optional: url to an external proxy storage app> 
+export GRAPHS_PROXY=<optional: url to an external proxy storage app>
 export USERS_LDAP_PROVIDER=<optional: ldap url for user lookup>
 
 ```
 note: see *Getting your google drive credentials.json file* below for the GD_CREDENTIALS value
 
 
-### Starting up the application
+#### Starting up the application
 
 Included in the pom is a jetty plugin, so as long as you have maven installed you just run:
 
 ```
 mvn clean package -DskipTests jetty:run -Djetty.port=8082
 ```
+### Local v2 Deployment
+
+#### Deploy Back-End Services
+
+1. Follow steps in https://github.com/redhat-cop/ninja-board/blob/v2/quarkus/README.MD
+
+#### Deploy Front-End
 
+1. Follow steps in https://github.com/redhat-cop/ninja-board/blob/v2/react/ninja-react/README.md
 
 ## Deployment on Openshift
 
-### Setting up the deployment config files
+### Deployment Using the OpenShift Applier
+
+The project can be deployed using the [openshift-applier](https://github.com/redhat-cop/openshift-applier).
+
+#### Prerequisites
+
+The following prerequisites must be satisfied:
+
+1. [Ansible](https://www.ansible.com/)
+2. OpenShift Command Line Interface (CLI)
+3. OpenShift environment
+
+### OpenShift v1 Deployment
+
```

Comment:
There is also a line with `Configure the mandatory deployment parameters in applier/params/community-ninja-board-deployment`. 

I know you did not add this line, but can you correct it to reference `.openshift/params/community-ninja-board-deployment`

Reply:
Changed to `.openshift/params/ninja-board-deployment`